winpcapy.py

```diff
@@ -140,3 +140,8 @@ def capture_on_and_print(cls, pattern):
         Usage example capture_on_and_print("*Intel*Ethernet") will capture and print packets from an Intel Ethernet device
         """
         cls.capture_on(pattern, cls.packet_printer_callback)
+
+    @staticmethod
+    def capture_on_dev_name(device_name, callback):
```

Comment:
While this is a nice convenience function, it doesn't really add any functionality that isn't provided by WinPcapUtils.capture_on() - since you can pass the full device name as part of a pattern (without any wildcards).
Perhaps it would be better to add a flag or logic to WinPcapUtils.capture_on() to avoid the device search, in the rare case where you really want to avoid the device search.